Cargo.toml

```diff
@@ -12,12 +12,6 @@ edition = "2018"
 [features]
 default = ["std"]
 
-# The NEON implementation does not participate in dynamic feature detection,
-# which is currently x86-only. If "neon" is on, NEON support is assumed. Note
-# that AArch64 always supports NEON, but support on ARMv7 varies. The NEON
-# implementation uses C intrinsics and requires a C compiler.
-neon = []
```

Comment:
If this change lands, something to keep in mind is that removing Cargo features is considered a breaking change and would require a `0.x` to `0.y` version bump. It might be better to release a backward-compatible version with NEON detection, and then later remove the flag when other breaking changes are happening.